### ISSUE TITLE
Code of conduct and contribution guidelines

### ISSUE THREAD

**Issue by MarcinAman** [ajambor91/weather-react#1]
I came across this awesome project but realised that you don't provide neither contribution guidelines nor code of conduct. In such a large project it is crucial to provide suggestions to contributors since for them it is a new environment.

I can give you a small template for that:
- [sample code of conduct](https://github.com/angular/angular/blob/master/CODE_OF_CONDUCT.md)
- [sample contribution guidlines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md)

You may also include the license as the project grows it will be essential

**Comment by Gryfit** [comment]
@MarcinAman good idea. As a to be contributor I feel discouraged by lack of basic guidlines, CoC and licence. Is crucial for any project that is to be maintained to have those.
